src/lib/util.ts

```diff
@@ -29,6 +29,14 @@ const download: () => void = () => {
   img.className = canvas.className;
   canvas.parentNode!.replaceChild(img, canvas);
 
+  //ボタンを見えなくする
+  const button = doc.getElementsByTagName("button");
+  for (let i = 0; i < button.length; i++) {
+    button[i].style.visibility = "hidden";
+  }
```

Comment:
添え字は特に使わないので `forEach` で回すほうがすっきりすると思います。
あと `display = "none"` にすることでもともとボタンがあった部分を詰めることができます。

```suggestion
  Array.from(doc.getElementsByTagName("button")).forEach(
    (e) => (e.style.display = "none")
  );
```